src/commands/datasets/get-items.ts

```diff
@@ -0,0 +1,146 @@
+import { Args, Flags } from '@oclif/core';
+import type { CustomOptions, FlagDefinition } from '@oclif/core/interfaces';
+import { type ApifyClient, type Dataset, type DatasetClient, DownloadItemsFormat } from 'apify-client';
+
+import { ApifyCommand } from '../../lib/apify_command.js';
+import { error, simpleLog } from '../../lib/outputs.js';
+import { getLocalUserInfo, getLoggedClientOrThrow } from '../../lib/utils.js';
+
+export class DatasetsGetItems extends ApifyCommand<typeof DatasetsGetItems> {
+	static override description = 'Exports the items present in a Dataset.';
+
+	static override flags = {
+		limit: Flags.integer({
+			description:
+				'The amount of elements to get from the dataset. By default, it will return all available items.',
+		}),
+		offset: Flags.integer({
+			description: 'The offset in the dataset where to start getting items.',
+		}),
+		format: (
+			Flags.string as FlagDefinition<
+				DownloadItemsFormat,
+				CustomOptions,
+				{
+					multiple: false;
+					requiredOrDefaulted: false;
+				}
+			>
```

Comment:
This looks arcane. Would the enum options not pass the type check? Or what is going on? :smile:

Reply:
Its more so its strictly typed to not require a further cast (cuz ocliff isn't smart enough to handle choices as constants)

Reply:
Huh, it still freaks me out a bit, but thanks for the explanation. I guess a comment would help, especially if it explained why this is a safe thing to do.